platform/darwin/src/MGLAccountManager_Private.h

```diff
@@ -1,14 +1,21 @@
 #import "MGLAccountManager.h"
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I added nullability notations to `MGLAccountManager_Private.h` because they were missing, but we will need to double check that they’re correct and not harmful.